### ISSUE TITLE
Last Active Status Will Not Set to Private in Forums

### ISSUE THREAD

**Issue by TheObstinateNoviceSmith** [Nexus-Mods/web-issues#1072]
## Bug/error being reported:
I've gone to every option, asked people for help and followed their instructions, and spent hours looking for myself.  No matter what, on the forum side of things, the last time I was on is always displayed.

This is my second time posting an issue here and I'm not even sure if I'm posting in the right section because it seems that which problems get any acknowledgement here are random.

**Comment by Pickysaurus** [comment]
As far as I know this isn't an option with our current forum software.

**Comment by TheObstinateNoviceSmith** [comment]
Did you all randomly decide to remove this feature?  Those people I asked how they did it; they're last active shows as "private" in the forums.

And if you did remove it, why close this instead of listing it as a "feature request" at least?

**Comment by Pickysaurus** [comment]
Could you provide an example? We allow this on the site side (https://www.nexusmods.com/users/myaccount), but do not plan to make any significant changes to the forum for the foreseeable future. 

The option does not exist here (https://forums.nexusmods.com/index.php?app=core&module=usercp) for me as an admin, so it's likely any users with it set private did so a long time ago - possibly on a previous version of the forum software.

**Comment by TheObstinateNoviceSmith** [comment]
So it was a feature (which is why profiles like Oubilette and Simdrew1993 show as private in the forums), but at some point it stopped being a feature, and there's no plans to bring the feature back?

And if that is true, why not?

**Comment by Pickysaurus** [comment]
I am unsure exactly when it changed, however, my best guess would be this was part of an older version of the IPBoard software and was not supported when we upgraded. Simdrew1993 registered in 2013 so this tracks. I have checked the admin area and I can not find any options relating to this. 

As I said, we have no plans on making changes to the forum for the foreseeable future.

Thanks for the feedback
